library/core/src/any.rs

```diff
@@ -1,5 +1,5 @@
-//! This module implements the `Any` trait, which enables dynamic typing
-//! of any `'static` type through runtime reflection.
+//! The `Any` trait enables dynamic typing of any `'static` type through runtime
+//! reflection.
```

Comment:
This module also provides `TypeId` and `type_name`. While related to `Any`, they're also useful without directly using that trait.

Not entirely sure how to summarize these together in one line though. Something about type erasure or dynamic typing or reflection maybe?

Reply:
How about

> Utilities for dynamic typing or working with any type.

supposing `Any` and `TypeId` falls under "dynamic typing" and `type_name` is a util that works with any type.

Reply:
Yeah that could work. Or maybe "Utilities related to type erasure."? Either way is fine, as long as it doesn't imply it's only the `Any` type.

Reply:
Currently landed on "Utilities for dynamic typing or type reflection." I feel that "erasure" is a sub-concept of "dynamic typing", and `TypeId` and `type_name` are both utilities for "reflection", so that word is good to include.